{{ cookiecutter.slug }}/README.md

```diff
@@ -3,15 +3,15 @@
 This is a [Commodore][commodore] Component for {{ cookiecutter.name }}.
 
 This repository is part of Project Syn.
-For documentation on Project Syn and this component, see https://syn.tools.
+For documentation on Project Syn and this component, see [syn.tools](https://syn.tools).
 
 ## Documentation
 
 The rendered documentation for this component is available on the [Commodore Components Hub](https://hub.syn.tools/{{ cookiecutter.slug }}).
 
 Documentation for this component is written using [Asciidoc][asciidoc] and [Antora][antora].
-It is located in the [docs/](docs) folder.
-The [Divio documentation structure](https://documentation.divio.com/) is used to organize its content.
+It can be found in the [`docs`](docs) folder.
+The [Divio documentation structure](https://documentation.divio.com/) organizes its content.
```

Comment:
IMO this change makes the sentence even more awkward

I'd reword this as something like

```suggestion
We use the [Divio documentation structure](https://documentation.divio.com/) to organize our documentation.
```

Reply:
Yes, that does sounds a bit better. 🙃